src/Text/__tests__/index.test.js

```diff
@@ -2,7 +2,7 @@
 
 import * as React from 'react';
 import { render } from 'react-native-testing-library';
-
+import snapshotDiff from 'snapshot-diff';
```

Comment:
What does this do exactly?

Reply:
TBH I see that Callstack use this so I want to try.
But I should helpful for testing the difference between different component states. 
I followed the pattern that I saw in our codebase. 
I think about like, that if you change something in the rendering based on the status, in the default or extended the component, the test falls and notice you.